src/ui/TemplateMessageItemBody/index.tsx

```diff
@@ -0,0 +1,73 @@
+import './index.scss';
+import React, { ReactElement } from 'react';
+import type { BaseMessage } from '@sendbird/chat/message';
+import { getClassName } from '../../utils';
+import MessageTemplateProvider from '../../modules/GroupChannel/components/MessageTemplateProvider';
+import { MessageTemplateData, MessageTemplateItem, SendbirdMessageTemplate } from './types';
+import restoreNumbersFromMessageTemplateObject from './utils/restoreNumbersFromMessageTemplateObject';
+import mapData from './utils/mapData';
+import selectColorVariablesByTheme from './utils/selectColorVariablesByTheme';
+import { SendbirdTheme } from '../../types';
+
+interface Props {
```

Comment:
How about `TemplateMessageItemBodyProps`? It think it would be better to use the specific interface name

Reply:
agreed ill push to child PR